tools/bin/ecflow/processes/main/test_date.ecf

```diff
@@ -7,7 +7,7 @@ Check date to determine whether we will run forecast update
 echo "Today is $(date)"
 day=$(date +%%-d)
 #TODO move day check/event to a separate task
-if [[ $day -eq 10 ]] ; then
+if [[ $day -eq 11 ]] ; then
```

Comment:
I think that would be great, but I'm not 100% sure how to implement it. I think if we try to use an event for triggering, it would block the system from requeuing. Maybe we could have a task that is triggered by the download aborting that sets a variable in the suite, which would then be available for checking in `test_date.ecf`